sdk/cosmos/azure-cosmos/azure/cosmos/aio/_cosmos_client.py

```diff
@@ -265,6 +267,7 @@ async def create_database(
         :paramtype offer_throughput: Union[int, ~azure.cosmos.ThroughputProperties]
         :keyword dict[str, str] initial_headers: Initial headers to be sent as part of the request.
         :keyword response_hook: A callable invoked with the response metadata.
+        :keyword int throughput_bucket: The desired throughput bucket for the client
```

Comment:
It should be added to the per-method documentation that a value specified here will override any value already specified at the client-level.

Reply:
Also - it looks like this was maybe copy+pasted from the client-level documentation? This should be rephrased as it's not the bucket "for the client", rather it's the bucket "for the request" (or maybe "operation", as some of these, like query, are multiple requests).